service/models.py

```diff
@@ -37,7 +37,6 @@ def deserialize(self, data: dict):
 
         :param data: a Python dictionary representing an Item.
         """
-        print("deserialize(%s)", data)
```

Comment:
can you add this back as a `logger.info` call? might be useful for debugging purposes.

Reply:
Done